### ISSUE TITLE
Feature Request: Show a small tab/label of the version/translation title right at the top left or right of the window

### ISSUE THREAD

**Issue by openbibles** [rubiot/ibiblia#8]

**Comment by rubiot** [comment]
Well, I guess this is a problem only when you're starting a new project. After you've settled on the texts you'll use you'll be working with them for months. Because of that I'm giving this a low priority, OK?

**Comment by rubiot** [comment]
![image](https://user-images.githubusercontent.com/8127450/77382670-b25d4880-6d5f-11ea-8801-4aba7bf4dffc.png)
Added fields to enter a title/description for each text in the Project Properties dialog. I don't think it's necessary to add something to the main screen for the reasons I gave before: the kind of work that is done in iBiblia will take months and any eventual confusion about the texts shouldn't last for more than a couple of days. Anyway, the properties dialog can be easily brought up with the shortcut Ctrl+P in case one needs to confirm this information.

This description is filled automatically with data from the modules you import (copied from "description" property) at project creation or later using the "Load new text" feature. It can be changed manually at any time though.
